bin/blockfrostUtils.js

```diff
@@ -0,0 +1,74 @@
+
+import { readFile } from 'fs/promises';
+import { BlockFrostAPI } from "@blockfrost/blockfrost-js";
+
+/**
+ * 
+ * @param {*} blockfrostKey 
+ * @param {*} walletAddress 
+ * @returns 
+ */
+const queryUTxOviaBlockfrost = async (blockfrostKey, walletAddress) => {
+  const blockfrost = new BlockFrostAPI({
+    projectId: blockfrostKey,
+  });
+  const utxos = await blockfrost.addressesUtxosAll(walletAddress);
+  let TxHash = "";
+  let TxIx = 0;
+  let Amount = 0;
+  for (const utxo of utxos) {
+    if (utxo.amount.reduce((sum, item) => sum + parseInt(item.quantity, 10), 0) > 1000000) {
```

Comment:
Same here, use chain operator so should look like `item?.quantity`, `utxo?.amount`, `utxo?.tx_hash`, `utxo?.output_index`